### ISSUE TITLE
Allow coach/manager to login with their email but manage all their players profile through it.

### ISSUE THREAD

**Issue by Joe2k** [india-ultimate/hub#148]
Allow coach/manager to login with their email but manage all their players profiles through it but not needing to have multiple email/ phone numbers. For ex. Awadh, and other school teams

**Comment by punchagan** [comment]
I think it would be good to discuss and elaborate upon the implementation, before getting started on this.

**Comment by Joe2k** [comment]
yep agreed, and also would be a low priority issue right now. Good to target for next season only

**Comment by SibiAkkash** [comment]
good feature to pick up
